index.php

```diff
@@ -1,14 +1,18 @@
 <?php
 
-require __DIR__ . DS . "src" . DS . "Autopublish.php";
+use Kirby\Cms\Response;
```

Comment:
You returned `return new Response('done', 'text/html');`  but forgot to import `Response`.